includes/rtm/impl/vector4f_swizzle.h

```diff
@@ -0,0 +1,193 @@
+#pragma once
```

Comment:
Good idea to move this into its own header.
As a convention, all similar headers have the suffix `.impl.h`
The header is also missing the MIT license information, see other headers as example.